packages/entity-server/src/__tests__/types.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import {
+  ModelRegistry,
+  EntityModel,
+  EntityHierarchyModel,
+  EntityRelationModel,
+  UserEntityPermissionModel,
+  UserModel,
+  ProjectModel,
+} from '@tupaia/database';
+
+export interface TestModelRegistry extends ModelRegistry {
+  readonly entity: EntityModel;
+  readonly entityHierarchy: EntityHierarchyModel;
+  readonly entityRelation: EntityRelationModel;
+  readonly user: UserModel;
+  readonly userEntityPermission: UserEntityPermissionModel;
+  readonly project: ProjectModel;
+}
```

Comment:
I don't mind ModelRegistry being an opt-in kind of thing, since it helps with information hiding! But it would be really cool for it to be generic or something, eg. `type TestModelRegistry = ModelRegistry<[entity, entityHierarchy, entityRelation, user, userEntityPermission, project]>`